analysis/system-info.js

```diff
@@ -3,13 +3,15 @@
 class SystemInfo {
   constructor (data) {
     this.providers = new Set(data.providers)
-    this.pathSeperator = data.pathSeperator
     this.mainDirectory = data.mainDirectory
 
+    // Backwards compatibility with data collected in Bubbleprof <1.6.1, which contained a typo
+    this.pathSeparator = data.pathSeparator || data.pathSeperator
```

Comment:
the SystemInfo class is internal. the `data` variable here refers to the JSON data in the `PID.clinic-bubbleprof-systeminfo` file, so it might be from an older version.